metadetect/mfrac.py

```diff
@@ -0,0 +1,49 @@
+import ngmix
+import numpy as np
+
+from .shearpos import DEFAULT_STEP
+from .detect import CatalogMEDSifier
+from .defaults import BMASK_EDGE
+
+
+def measure_mfrac(
```

Comment:
can you please add some docs here, and also explaining what it is you are doing?

Reply:
🤦 Sorry!

Reply:
done!